WalletWasabi.Gui/UiConfig.cs

```diff
@@ -44,10 +44,10 @@ public class UiConfig : ReactiveObject, IConfig
 		public bool? Autocopy { get; internal set; }
 
 		[JsonProperty(PropertyName = "LurkingWifeMode")]
-		public bool? LurkingWifeMode
+		public bool LurkingWifeMode
```

Comment:
Changes based on that you removed the nullable from LurkingWifeMode. It is all about patterns so as @nopara73 said we cannot change this.